src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp

```diff
@@ -267,7 +267,7 @@ class KerrSchild : public AnalyticSolution<3_st>,
       "Black hole in Kerr-Schild coordinates"};
 
   KerrSchild(double mass, const std::array<double, 3>& dimensionless_spin,
-             const std::array<double, 3>& center,
+             const std::array<double, 3>& center = {{0., 0., 0.}},
```

Comment:
Why do we need a default? In general we've had a theme of minimizing defaults to increase transparency.

Reply:
The center seems like an optional thing to move away from zero and constructing a Kerr solution as `KerrSchild{mass, spin}` seems rather obvious, so I think the default makes sense here. The boost velocity also has a default value (zero) for a similar reason.

Reply:
I have a slight preference, but would like Mark & Will's input. Maybe ping them on Slack to see how they feel? Mark write KS, I think.